build.sh

```diff
@@ -44,12 +44,20 @@ function flatten() {
   docker rm $container
 }
 
-echo "Building image conjur:$TAG"
-docker build -t "conjur:$TAG" .
-flatten "conjur:$TAG"
+image_not_built() {
+  [[ "$(docker images -q "$1" 2> /dev/null)" == "" ]]
+}
+
+if image_not_built "conjur:$TAG"; then
```

Comment:
There's a risk here that attempting to rebuild the image locally after change the code won't work unless the TAG (namely git commit hash) has also changed.

We may want to explicitly log if we are skipping the build because the image is up to date to make this more obvious at a glance.

Reply:
Another option could be to incorporate the `--jenkins` flag into this condition, so this primarily affects the CI nodes, rather than local dev (where `build.sh` happens less frequently and more intentionally).

Reply:
Great point.  I like the latter suggestion.